datacenter/dtr/2.6/guides/user/manage-images/override-a-vulnerability.md

```diff
@@ -5,31 +5,24 @@ description: Learn how to dismiss a vulnerability reported by the security
 keywords: registry, security scanner
 ---
 
-DTR scans your images for vulnerabilities but sometimes it can report that
-your image has vulnerabilities you know have been fixed. If that happens you
-can dismiss the warning.
+DTR scans images for vulnerabilities. At times, however, it may report image vulnerabilities that you know have been fixed, and whenever that happens the warning can be dismissed.
 
```

Comment:
Add a sentence to introduce this procedure.

Reply:
Opting to leave it as-is and let the heading serve as the lead-in.